package.json

```diff
@@ -105,14 +106,14 @@
     "typescript": "^5.0.4",
     "webpack": "^5.90.3"
   },
-  "packageManager": "pnpm@8.15.6",
+  "packageManager": "pnpm@9.4.0",
```

Comment:
Seems like a change that should be done in a separate PR?

Also, latest pnpm version right now is 9.10.0

If you needed these changes to be able to work locally, consider simply undoing them again before merging the PR.